test.scf

```diff
@@ -36,8 +36,15 @@ run without options
 
 cover including commit messsages (smoke test)
     $ git-big-picture -c -g >/dev/null
+
+test option precedence for -C followed by -c (include commit messages)
+    $ git-big-picture -g -C -c | grep 'foo'
```

Comment:
This line is expected to produce at least one line of output in a real shell session. So I would vote for (a) passing `-q` to grep or for (b) adding a line like `[RE] foo` to communicate that we do expect output. What do you think?

Reply:
yeah, I was surprised to see this pass, will add `-q` to be safe.

Reply:
done